ct-aa/dotTheory---TypeScript
# JS-973: Select and share a place app

# Related Tickets
[JS-973](https://codeandtheory.atlassian.net/browse/JS-973)

# Description
- Added the code for the select and share a place app. But it is not working right now because I didn't proceed to the billing stage of the Google Maps API.

# Validation
- Open the terminal.
- Run cd "select-and-share-a-place".
- Run "npm install".
- Run "npm start".
- Open the browser and check the local host.
- The browser should display the map container and input field for the app. But it won't work because I didn't proceed to the billing stage of the Google Maps API.

<img width="1787" alt="Screen Shot 2024-01-24 at 10 36 54 AM" src="https://github.com/ct-aa/dotTheory---TypeScript/assets/93116252/e078566d-d6b9-45e4-a53d-0b82848fa9fa">

# Risks
N/A
